config/validation.go

```diff
@@ -0,0 +1,66 @@
+package config
+
+import (
+	"fmt"
+	"github.com/contiv/errored"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+
+	gojson "github.com/xeipuuv/gojsonschema"
+)
+
+// Combines array of errors into a single error
+func combineErrors(resultErrors []gojson.ResultError) error {
```

Comment:
use `errored.Combine` here please and the `errors` package to prefix it with a canned error.